contracts/integrations/test/internal-integration-tests/match_orders_test.ts

```diff
@@ -0,0 +1,2754 @@
+import { BlockchainBalanceStore, TokenIds } from '@0x/contracts-exchange';
```

Comment:
Since we're already porting this code, I would be in favor of splitting it up by function `matchOrders`, `matchOrdersWithMaximalFill`, `batchMatch`, `batchMatchWithMaximalFill`.

Reply:
Btw the code in here is super clean :mister-clean:

Reply:
Are you suggesting that we should have `match_orders_test`, `match_order_with_maximal_fill_test`, etc? I'd be up for that.

Reply:
Yeah. I think cutting this down in size + splitting those out would go a long way wrt readability.